### PR TITLE
Update Python functions infra & configure Atlas

### PR DESCRIPTION
**Pull request by logachev**
- Add scripts to upload required typedefs and entities to Atlas
- Update Python functions infra

### CODE REVIEW

File: terraform/base-infra/outputs.tf [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 }
 
 output "base_acr_name" {
-  value = "${azurerm_container_registry.this.name}"
+  value = "\"${azurerm_container_registry.this.name}\""
 }
 
 output "sql_db_connection_string" {
-  value = "Server=tcp:${azurerm_sql_server.base.name}.database.windows.net,1433;Initial Catalog=${azurerm_sql_database.base_db.name};Persist Security Info=False;User ID=${var.sql_username};Password=${var.sql_password};MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
+  value = "\"Server=tcp:${azurerm_sql_server.base.name}.database.windows.net,1433;Initial Catalog=${azurerm_sql_database.base_db.name};Persist Security Info=False;User ID=${var.sql_username};Password=${var.sql_password};MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;\""
```
Comment (aluong) [inline terraform/base-infra/outputs.tf]: Why do we need these extra quotes?

File: terraform/metadata-infra/functions.tf [hunk 2]
```diff
@@ -79,24 +80,3 @@ module "lineage_creator" {
     sql_password = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.SqlPassword.id})"
   }
 }
-
-module "json_generator" {
```
Comment (aluong) [inline terraform/metadata-infra/functions.tf]: Is this not an App needed anymore?

Comment (logachev) [reply terraform/metadata-infra/functions.tf]: yeah, json_generator is a part of metadata_utility_services

File: terraform/base-infra/outputs.tf (hunk 1 above)
Comment (logachev) [reply terraform/base-infra/outputs.tf]: That way you can copy-paste from outputs to the 'tfvars' file

**Pull request merged**